bolt-cli/Cargo.toml

```diff
@@ -1,30 +1,42 @@
 [package]
-name = "bolt-delegations-cli"
+name = "bolt-cli"
```

Comment:
Can we rename this to just `bolt`? This will be the canonical CLI package, users would just have to invoke `bolt operator register ...` etc

Reply:
Agreed! Please remember to change the invocations of `bolt-cli` to `bolt` in the README then

Reply:
done!